x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/index/store/SearchableSnapshotIndexInput.java

```diff
@@ -61,17 +60,21 @@
     private static final long NO_SEQUENTIAL_READ_OPTIMIZATION = 0L;
 
 
-    SearchableSnapshotIndexInput(final BlobContainer blobContainer, final FileInfo fileInfo, long sequentialReadSize, int bufferSize) {
-        this("SearchableSnapshotIndexInput(" + fileInfo.physicalName() + ")", blobContainer, fileInfo, 0L, 0L, fileInfo.length(),
+    SearchableSnapshotIndexInput(
```

Comment:
I'll revert this spotless formatting

Reply:
I for one welcome our new spotless overlords :ant: